tests/Unit/Domain/Test_InterfaceItems.cpp

```diff
@@ -475,6 +486,172 @@ void test_interface_slice(){
+      db::AddComputeTags<
+          Tags::InterfaceCompute<Directions<Dim>, Tags::Direction<Dim>>,
+          Tags::InterfaceCompute<Directions<Dim>, Tags::InterfaceMesh<Dim>>,
+          Tags::InterfaceCompute<Directions<Dim>,
+                                 Tags::BoundaryCoordinates<Dim, true>>>>(
+      mesh, get_directions<Dim>(),
+      ElementMap<Dim, Frame::Grid>(logical_to_grid_map.element_id(),
+                                   logical_to_grid_map.block_map().get_clone()),
+      grid_to_inertial_map->get_clone(), time,
+      clone_unique_ptrs(functions_of_time));
+  for (const auto& direction_and_coordinates :
+       db::get<Tags::Interface<Directions<Dim>,
+                               Tags::Coordinates<Dim, Frame::Inertial>>>(box)) {
+    const auto& direction = direction_and_coordinates.first;
+    CHECK_ITERABLE_APPROX(
+        (*logical_to_inertial_map)(
+            interface_logical_coordinates(
+                mesh.slice_away(direction.dimension()), direction),
+            time, functions_of_time),
+        direction_and_coordinates.second);
+  }
+}
+
+void test_boundary_coordinates_moving_mesh() {
+  MAKE_GENERATOR(generator);
+  std::uniform_real_distribution<> dist(-1., 1.);
+
+  const std::array<double, 4> times_to_check{{0.0, 0.3, 1.1, 7.8}};
+  const double outer_boundary = 10.0;
+  std::array<std::string, 2> functions_of_time_names{
+      {"ExpansionA", "ExpansionB"}};
+  using Polynomial = domain::FunctionsOfTime::PiecewisePolynomial<2>;
+  using FoftPtr = std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>;
+  std::unordered_map<std::string, FoftPtr> functions_of_time{};
+  const std::array<DataVector, 3> init_func_a{{{1.0}, {-0.1}, {0.0}}};
+  const std::array<DataVector, 3> init_func_b{{{1.0}, {0.0}, {0.0}}};
+  const double initial_time = 0.0;
+  functions_of_time["ExpansionA"] =
+      std::make_unique<Polynomial>(initial_time, init_func_a);
+  functions_of_time["ExpansionB"] =
+      std::make_unique<Polynomial>(initial_time, init_func_b);
+
+  {
+    INFO("1d");
+    const ElementMap<1, Frame::Grid> logical_to_grid_map(
+        ElementId<1>(0), make_coordinate_map_base<Frame::Logical, Frame::Grid>(
+                             CoordinateMaps::Affine(-1.0, 1.0, 2.0, 7.8)));
+    const auto grid_to_inertial_map =
+        make_coordinate_map_base<Frame::Grid, Frame::Inertial>(
+            CoordMapsTimeDependent::CubicScale<1>{outer_boundary,
+                                                  functions_of_time_names[0],
+                                                  functions_of_time_names[1]});
+    const auto logical_to_inertial_map =
+        make_coordinate_map_base<Frame::Logical, Frame::Inertial>(
+            CoordinateMaps::Affine(-1.0, 1.0, 2.0, 7.8),
+            CoordMapsTimeDependent::CubicScale<1>{outer_boundary,
+                                                  functions_of_time_names[0],
+                                                  functions_of_time_names[1]});
```

Comment:
Here (and in 2d, 3d cases below), do you think it would be reasonable to structure things like this?
```cpp
const ... logical_to_grid_func = CoordinateMaps::Affine(-1.0, 1.0, 2.0, 7.8);
const ... grid_to_inertial_func = CoordMapsTimeDependent::CubicScale<1>{ ... };
const ElementMap<1, Frame::Grid> logical_to_grid_map(..., make_coordinate_map_base<...>(logical_to_grid_func);
const auto grid_to_inertial_map = make_coordinate_map_base<...>(grid_to_inertial_func);
const auto logical_to_inertial_map = make_coordinate_map_base<...>(logical_to_grid_func, grid_to_inertial_func);
```

The goal would be to avoid having to specify each coord map two separate times... But this does add more lines of code, so it's up to you whichever you think is the cleanest approach 🙂

Reply:
@nilsdeppe as a sanity check ... I see you updated the 2d and 3d map tests per this suggestion, but not the 1d test. Was this intentional? Or did you miss the 1d case?

Reply:
Nah, I'm just dumb :P